docs/user/openstack/install_upi.md

```diff
@@ -270,6 +271,22 @@ api.openshift.example.com.    A 203.0.113.23
 
 They will need to be available to your developers, end users as well as the OpenShift installer process later in this guide.
 
+## Network identifier
+
+Resources like network, subnet (or subnets), router and API and ingress ports need to have unique name to not interfere with other deployments running on the same OpenStack cloud.
+Let's create environment variable `OS_NET_ID` and `netid.json` file, which will be used by ansible playbooks later on.
+
+<!--- e2e-openstack-upi: INCLUDE START --->
+```sh
+$ export OS_NET_ID="openshift-$(dd if=/dev/urandom count=4 bs=1 2>/dev/null |hexdump -e '"%02x"')"
+$ echo "{\"os_net_id\": \"$OS_NET_ID\"}" | tee netid.json
+```
+<!--- e2e-openstack-upi: INCLUDE END --->
+
+Make sure your shell session has the `$OS_NET_ID` environment variable set when you run the commands later in this document.
+
+Note, this identifier has nothing in common with OpenShift `infraID` defined later on.
```

Comment:
Have you checked if it is possible to force `infraID` to be identical to the generated `os_net_id`? Otherwise the networking resources would look like they belong to a different cluster in UPI and it's a possible source of confusion.

Reply:
Yes I did. There is no way for passing any arbitrary `infraID` nor `clusterID` into `install-config.yaml`. The `infraID` is generated during ignition creation.